test/e2e/onINP-test.js

```diff
@@ -841,6 +841,54 @@ describe('onINP()', async function () {
 
       const [inp1] = await getBeacons();
       assert(inp1.attribution.longAnimationFrameEntries.length > 0);
+      assert(inp1.attribution.longAnimationFrameSummary != {});
```

Comment:
This assertion will never fail because the `longAnimationFrameSummary` object will never be the same object as an object you create inline via `{}`.

Reply:
Of course. What's the easiest way to test it's not empty. Like this?

```suggestion
      assert(Object.keys(inp1.attribution.longAnimationFrameSummary.length) !== 0);
```

Reply:
Yeah, but make sure to call `.length` of the result of the `Object.keys()` call.

```suggestion
      assert(Object.keys(inp1.attribution.longAnimationFrameSummary).length !== 0);
```